README.md

```diff
@@ -61,8 +61,16 @@ Visiting `http://localhost:3000` will allow you to `Modify scenarios`. The defau
 
 ## API
 
+### initialise
+
+returns the internal express instance
+
+> `function({ default, scenarios, options })`
+
 ### run
 
+returns the internal express server, with an additional kill method
```

Comment:
Can you change to:

```
Returns an http server, with an additional kill method
```

The fact that it uses express to build that server is irrelevant in this case.